policy/scan/local_scanner.go

```diff
@@ -261,7 +272,15 @@ func (s *LocalScanner) RunAssetJob(job *AssetJob) {
 					log.Warn().Err(err).Msg("failed to query platform information")
 				} else {
 					job.Asset.Platform = p
-					// resyncAssets = append(resyncAssets, assetEntry)
+					if upstream != nil {
+						_, err := upstream.SynchronizeAssets(job.Ctx, &policy.SynchronizeAssetsReq{
```

Comment:
is there a more minimal version of synchronise assets that we could call?

Reply:
Not that I could find. The goal is to have this temporarily until we start using MQL to retrieve platform information. That's why I added this as a quick fix until we plan out the details for the long term solution.

Reply:
do we need to call synchornize assets twice for each asset? can we call it once after we have the connection and collected the platform information?